Sources/PointFreeTestSupport/PointFreeTestSupport.swift

```diff
@@ -451,8 +451,13 @@ extension Session {
     |> \.userId .~ Database.User.mock.id
 }
 
-public func request(to route: Route, session: Session = .loggedOut, basicAuth: Bool = false) -> URLRequest {
-  var request = router.request(for: route, base: URL(string: "http://localhost:8080"))!
+public func request(
+  with baseRequest: URLRequest,
+  session: Session = .loggedOut,
+  basicAuth: Bool = false
+  ) -> URLRequest {
```

Comment:
this is like the `request(to:session:basicAuth:)` helper except it takes in a request instead of a route. this is helpful for when you want to very explicitly test a URL instead of just a route value.